packages/create/bin/create-stacks-workspace.ts

```diff
@@ -191,24 +190,18 @@ async function determineE2eTestRunner(
         });
 }
 
-async function getConfiguration(
+export async function getConfiguration(
     argv: yargs.Arguments<CreateStacksArguments>,
 ): Promise<void> {
     try {
         const name = await determineRepoName(argv);
         let { preset, appName, e2eTestRunner } = argv;
 
-        if (!preset) {
-            preset = await determinePreset(argv);
-        }
+        preset = await determinePreset(argv);
```

Comment:
do we no longer need any preset checks?

Reply:
determinePreset should always be called regardless of whether preset has a value or not. All the necessary logic is contained within the determinePreset function, to handle whether preset is set or not, and if it is set if that value is valid.